src/lib/settings/SchemaPiece.js

```diff
@@ -9,7 +9,7 @@ const fs = require('fs-nextra');
 class SchemaPiece extends Schema {
 
 	/**
```

Comment:
Missing docs for typedef AddOptions.

Reply:
Those were typos. The typedef is called `SchemaFolderAddOptions`.